docs/admin/addons.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I am thinking what makes most sense is to split the definition of add-ons into "built-in add-ons" and "3rd party add-ons". The former will obey the semantics in the [cluster addons README](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons) and the latter will be "just `kubectl apply -f` it" style external add-ons like Weave Net. How does that sound @caseydavenport?